### DISCUSSION TITLE
Save projects table view collapsed state

### DISCUSSION THREAD

**Discussion by hamidfzm** [community/community#17340] [Projects and Issues]
Recently I used the scrum method on Github Projects. On my Backlog view, currently, I have multiple sprints.  I don't need to view ended sprint tasks all the time. So I need them collapsed.  Currently, I have 3 sprints but imagine when I have 30 sprints like in my previous project. I have to scroll all day to view my No Sprint tasks or a specific sprint.

<img width="792" alt="Screen Shot 1401-03-06 at 03 44 12" src="https://user-images.githubusercontent.com/5591800/170594262-76d5e066-adef-456d-bb23-618fbbf2458f.png">

**Comment by roel4ez** [comment]
Came here to suggest this. We are dealing with the same issue, and I don't want to remove closed items from my board, since we are using the Insights to generate charts.
